expyfun/visual/_visual.py

```diff
@@ -1086,14 +1116,25 @@ def draw(self):
         self._player.update_texture()
         # detect end-of-stream to prevent pyglet from hanging:
         if not self._eos:
-            self._texture = self._player.get_texture()
-            self._scale_texture()
-            self._texture.blit(*self._actual_pos)
+            if self._visible:
+                self._texture = self._player.get_texture()
+                self._scale_texture()
+                self._texture.blit(*self._actual_pos)
         else:
             self._finished = True
             self.pause()
         self._ec.check_force_quit()
 
+    def set_visible(self, show, flip=False):
```

Comment:
if `flip=False` by default, is there much advantage to:
```
video.set_visible(True, flip=True)
```
over
```
video.set_visible(True)
ec.flip()
```
? I guess it saves a line, is that the idea?